src/stream/src/common/table/state_table.rs

```diff
@@ -162,6 +162,8 @@ pub struct StateTableInner<
     op_consistency_level: StateTableOpConsistencyLevel,
 
     clean_watermark_index_in_pk: Option<i32>,
+
+    on_post_commit: bool,
```

Comment:
Please leave some comment for this field, just like those in the PR description.